testing/trino-plugin-reader/src/test/resources/server-plugins/.gitignore

```diff
@@ -0,0 +1 @@
+
```

Comment:
Why do we need this empty file?

Reply:
The test in this module makes sure the whole plugin reader runs successfully, and it points to this empty dir to read plugins from it. But I didn't want to add real JARs here that could be actually loaded as a plugin, I'm not sure how to create the smallest possible plugin JAR like this.

Reply:
How about changing this to a comment that explains why this file exists?

Reply:
that could be a README file then

Reply:
Done, and rebased after 373 got released!